src/basics/behavior.jl

```diff
@@ -131,11 +131,10 @@ end
 
 render(t::Selectable, state) =
     render(t.tile, state) <<
-        Elem("selectable-behavior",
+        Elem("selectable-behavior", multi = t.multi,
             attributes = @d(
                 :name=>t.name,
-                :selector=>t.selector,
-                :multi=>t.multi
```

Comment:
The right way to fix this is to change this line to `:multi=>boolattr(t.multi)` right here. moving it out from attributes will cause updates to not trigger "multi-changed" event (assuming there is one).